tensorflow/compiler/xla/service/gpu/alpa_nccl_wrapper.cc

```diff
@@ -246,15 +264,16 @@ StatusOr<int> NcclGetVersion() {
 #endif  // XLA_ENABLE_XCCL
 }
 
-StatusOr< std::shared_ptr< std::vector<ncclComm_t> > > NcclCreateCommunicators(int world_size, 
-                                                                               std::vector<int> devices_global_rank, 
-                                                                               std::vector<int> devices_ids, 
-                                                                               std::vector<char> nccl_uid_vec) {
+StatusOr< std::shared_ptr<NcclCommStorage> > NcclCreateCommunicators(int world_size, 
```

Comment:
Code style:
```
StatusOr<std::shared_ptr<NcclCommStorage>> NcclCreateCommunicators(int world_size, 
```

Reply:
OK, I will create another PR to delete cupy nccl later. I will modify what you mentioned here along with that PR.